ads/google/a4a/utils.js

```diff
@@ -205,10 +204,10 @@ export function extractGoogleAdCreativeAndSignature(
             responseHeaders.get(AMP_SIGNATURE_HEADER)));
     }
     if (responseHeaders.has(CREATIVE_SIZE_HEADER)) {
-      const sizeStr = responseHeaders.get(CREATIVE_SIZE_HEADER);
-      // We should trust that the server returns the size information in the
-      // form of a WxH string.
-      size = sizeStr.split('x').map(dim => Number(dim));
+      const sizeArr = responseHeaders.get(CREATIVE_SIZE_HEADER)
+          .split('x')
+          .map(dim => Number(dim));
+      size = {width: sizeArr[0], height: sizeArr[1]};
```

Comment:
Depends on how malformed the returned string is. If it's something that can be split by x, things might still render. If it's complete gibberish, we'll end up with [NaN], and throw an out of bounds exception on L210. Should we perform additional checks here? And if they fail, what should we do?